prep-javascript-functions/main.js

```diff
@@ -0,0 +1,78 @@
+function addTwoNumbers_(x, y) {
+  var result = x + y;
+  return result;
+}
+
+var sum_ = addTwoNumbers_(4, 4);
+console.log('sum_', sum_);
+
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+
+var sum = addTwoNumbers(2, 2);
+console.log('sum', sum);
+
+function convertHoursToMinutes(hours) {
+  var result = hours;
+  return result;
+}
+
+var Minutes = convertHoursToMinutes(2 * 60);
+console.log(Minutes);
```

Comment:
For this one we want to move the multiplication by 60 into the function's code block.